WireDomain/Sources/WireDomain/Repositories/TeamRepository.swift

```diff
@@ -236,9 +236,9 @@ final class TeamRepository: TeamRepositoryProtocol {
 
 }
 
-private extension ConversationAction {
+extension ConversationAction {
 
-    var name: String {
+    fileprivate var name: String {
```

Comment:
Hmm when I write code I always prefer this `private extension`, because it means automatically nothing is available outside this file which makes it easier to refactor etc.
Now with the scope next to the property, one could just add a property `public var displayName: String`, but then I would move extension to a new file to have it clear.

Is it a special case here or shall we modify the rule?